### ISSUE TITLE
Validation dataset split

### ISSUE THREAD

**Issue by YuhanLiu11** [xuyige/BERT4doc-Classification#9]
Hi,
Thanks so much for sharing the code for this fantastic work! 
In the paper you mentioned that "We empirically set the max number of the epoch to 4 and save the best model on the validation set for testing". I am wondering how did you create the validation dataset for the classification tasks? Did you split the original train dataset into train/val? If that's the case, what's the ratio you split train/validation dataset for the IMDB, AGnews etc.? 

Thanks so much for your help in advance!

**Comment by xuyige** [comment]
Sorry for answering your issue so late.
We did not split the original train dataset into train/validation but directly evaluate our model on the test set.
However, we also tried to split 0.1 into validation and get a similar `test error rate`.

**Comment by lyh1121** [comment]
Thank you for your reply!

**Comment by YuhanLiu11** [comment]
Thank you very much for your reply!
